source/widgets/library_widget.py

```diff
@@ -342,7 +343,14 @@ def context_menu(self):
 
     @pyqtSlot(bool)
     def update_delete_action(self, shifting: bool):
-        if shifting:
+        reverted_behavior = False
+
+        if get_default_delete_action() == 1:
+            reverted_behavior = True
+
+        delete_from_drive = not reverted_behavior if shifting else reverted_behavior
```

Comment:
Sorry if I bother you, but I wanted to notify this affects the text of deleteAction, but the actual functionality has not yet changed in: https://github.com/Victor-IX/Blender-Launcher-V2/blob/6fe52a320235731fdc180bfece20415e99ecabd0/source/widgets/library_widget.py#L534-L554

Reply:
My bad. I forgot this part of the implementation😅